src/components/Carousel.js

```diff
@@ -0,0 +1,59 @@
+import carousel from "../styles/carousel";
+import carouselItems from "../assets/carousel-slides/carouselItemsData";
+import { useState } from "react";
+
+import ForwardIcon from "@material-ui/icons/Forward";
+
+const carouselItemsCount = carouselItems.length - 1;
+
+const Carousel = () => {
+  const carouselStyles = carousel();
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  const showNextSlide = () => {
+    if (currentSlide < 2) {
+      setCurrentSlide(currentSlide + 1);
+    } else {
+      setCurrentSlide(0);
+    }
+  };
+
+  const showPreviousSlide = () => {
+    if (currentSlide > 0) {
+      setCurrentSlide(currentSlide - 1);
+    } else {
+      setCurrentSlide(carouselItemsCount);
+    }
+  };
+
+  return (
+    <div className={carouselStyles.carousel}>
+      <div
+        className={carouselStyles.carouselInner}
+        style={{ backgroundImage: `url(${carouselItems[currentSlide].img})` }}
```

Comment:
I know it's faster and easier to add style, but more scalable will be adding classNames, easier for future to add/remove styles

Reply:
I kept inline for bg img only - it depends on the current state - other styling is in carouselInner class.